examples/quickstart/simple-coffee-machine.js

```diff
@@ -99,6 +106,14 @@ servient.start().then((WoT) => {
                         waterAmount = waterAmount - 10;
                         beansAmount = beansAmount - 10;
                         thing.emitPropertyChange("resources");
+                        if (waterAmount <= 10) {
+                            thing.emitEvent("resourceEmpty", "water");
+                            return undefined;
```

Comment:
@egekorkan do you men this return is needed even though it follows also afterwards?

Reply:
So when I don't do the return after an event emission and the conditions are met to fire two events, the two events are fired at the same time.

Reply:
I see. What I wanted to understand is whether you get *one* event with both or 2 events (which would be okay..).
I think what you are saying is that there is just 1 event, right?

Reply:
Yes 1 event from the Client side. Sorry, I also gave the same information below. We had a race condition ourselves here :D

Reply:
I have changed this part of code so I am resolving this. The actual problem is an issue now